redis/src/connection.rs

```diff
@@ -806,6 +806,23 @@ pub fn connect(
     setup_connection(con, &connection_info.redis)
 }
 
+pub(crate) fn client_set_info_pipeline() -> Pipeline {
```

Comment:
In other clients, we're:

1. Making this optional (but default on)
2. Allowing the variables used to be overrideable. 

The reason behind #2 is to allow folks building on top of a library, to embed a name within the string, for example: redis-rs(myfooclient)

Reply:
1. ATM we have pretty thin configurations. I'm not against doing this, but it will require a small paradigm change in how we configure our connections.
2. This I don't understand. The intention behind the CLIENT SETINFO command is explicitly to save *client* data, not application data - CLIENT SETNAME is usually used for that*. 

Since anyone can all CLIENT SETINFO and overwrite the previous value, I'm not eager to complicate our existing API in order to enable this, unless there's a concrete user request for that.

*the story I heard is that historically CLIENT SETINFO was intended to be used to save client data, but was co-opted by applications.